Description: Made changes to MakeFile
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 python3-huggingface-hub (0.29.3-1) UNRELEASED; urgency=medium
 .
   * Initial release. (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: anudeep <anudeep@unknown>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2025-03-24

--- python3-huggingface-hub-0.29.3.orig/Makefile
+++ python3-huggingface-hub-0.29.3/Makefile
@@ -7,33 +7,33 @@ check_dirs := contrib src tests utils se
 quality:
 	ruff check $(check_dirs)  # linter
 	ruff format --check $(check_dirs) # formatter
-	PYTHONPATH=$(CURDIR)/src python utils/check_inference_input_params.py
-	python utils/check_contrib_list.py
-	PYTHONPATH=$(CURDIR)/src python utils/check_static_imports.py
-	PYTHONPATH=$(CURDIR)/src python utils/check_all_variable.py
-	python utils/generate_async_inference_client.py
+	PYTHONPATH=$(CURDIR)/src python3 utils/check_inference_input_params.py
+	python3 utils/check_contrib_list.py
+	PYTHONPATH=$(CURDIR)/src python3 utils/check_static_imports.py
+	PYTHONPATH=$(CURDIR)/src python3 utils/check_all_variable.py
+	python3 utils/generate_async_inference_client.py
 
 	mypy src
 
 style:
 	ruff format $(check_dirs) # formatter
 	ruff check --fix $(check_dirs) # linter
-	python utils/check_contrib_list.py --update
-	PYTHONPATH=$(CURDIR)/src python utils/check_static_imports.py --update
-	PYTHONPATH=$(CURDIR)/src python utils/check_all_variable.py --update
-	python utils/generate_async_inference_client.py --update
+	python3 utils/check_contrib_list.py --update
+	PYTHONPATH=$(CURDIR)/src python3 utils/check_static_imports.py --update
+	PYTHONPATH=$(CURDIR)/src python3 utils/check_all_variable.py --update
+	python3 utils/generate_async_inference_client.py --update
 
 inference_check:
-	python utils/generate_inference_types.py
-	python utils/check_task_parameters.py
+	python3 utils/generate_inference_types.py
+	python3 utils/check_task_parameters.py
 
 inference_update:
-	python utils/generate_inference_types.py --update
-	python utils/check_task_parameters.py --update
+	python3 utils/generate_inference_types.py --update
+	python3 utils/check_task_parameters.py --update
 
 
 repocard:
-	python utils/push_repocard_examples.py
+	python3 utils/push_repocard_examples.py
 
 
 test:
